conmon-rs/server/src/file_logger.rs

```diff
@@ -0,0 +1,256 @@
+//! File logging functionalities.
```

Comment:
we should probably make clear this is the cri format, as json-file may one day be a supported type

Reply:
I changed the module to be `cri_logger.rs` does this fit better?